src/main/java/com/kodar/academy/Library/service/impl/UserServiceImpl.java

```diff
@@ -75,6 +79,10 @@ public UserResponseDTO updateUser(int id, UserEditDTO userEditDTO) {
 
     @Override
     public void deleteUser(int id) {
```

Comment:
This method executes multiple queries to the DB, so it would be best if it was Transactional